src/Microsoft.Graph.Core/Exceptions/ServiceException.cs

```diff
@@ -37,15 +37,5 @@ public bool IsMatch(string errorCode)
 
             return false;
         }
-
-        public override string ToString()
```

Comment:
I don't see how anyone can depend on this, what would that code look like? 

Exception ToString result show up in a number of places to help debug issues. Overriding it to return null or anything less than base exception's ToString breaks debugging process, as the runtime information, like call stack, is lost. What remains is an entirely unhelpful null, or a generic error message with no context information. It was impossible to debug the tests without removing this, which is why I made the change. I strongly recommend getting rid of the override and using the default exception behavior, which with the improved message, should always produce something meaningful now.